### DISCUSSION TITLE
The hightlight label for buffer filename as in screenshot

### DISCUSSION THREAD

**Discussion by puppyapple** [LazyVim/LazyVim#1986] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/33118185/22b4f361-259f-4cc5-980a-6da3ac8b53e4)
I just wondering which plugin provides the pink label top right as shown in the screenshot. I did not get this display.

**Comment by amaanq** [comment, marked as answer]
incline.nvim

**Reply by puppyapple** [reply]
thanks man!
